### DISCUSSION TITLE
在女武神潜艇出现的问题

### DISCUSSION THREAD

**Discussion by falldaylight** [FakeFishGames/Barotrauma#14985] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

在上层对接口出现损坏时，AI队友去进行维修时，会卡在梯子上动不了，无法进行任何指令。
![Barotrauma 2024_10_28 23_59_32](https://github.com/user-attachments/assets/86964835-3bf0-4946-a0d0-f12237b966f3)

### Reproduction steps

1,让女武神潜艇的对接口出现损坏2，向AI下达维修指令3，AI卡在梯子上

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.6.18.1 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by jigardi** [comment]
Machine translation of the report to English, in case it is useful for devs or anyone else:

"""
TITLE: Problems with the Submarine of the Goddess

**What happened?**
When the upper layer of the interface is damaged, when the AI teammate goes for maintenance, it will be stuck on the ladder and no instructions can be made. 

**Reproduction steps**
1, let the female Wushen submarine on the interface damage
2, to the AI to issue maintenance instructions 
3, AI stuck on the ladder

**Bug prevalence**
Happens regularly

**Single player or multiplayer?**
Single Player

--

_No response_

**Version**
v1.6.18.1 (Unto the Breach Update)

--

_No response_

**Which operating system did you encounter this bug on?**

Windows

**Relevant error messages and crash reports**

_No response_
"""
Edit123: Tried to adhere to the original formatting as much as possible, but made changes to repro-steps formatting so that it displays as a list.

**Reply by SomeRandomNoobKekeke** [reply]
It's been addressed https://github.com/FakeFishGames/Barotrauma/discussions/15261

**Reply by jigardi** [reply]
Thanks for the notification! : D

**Comment by Regalis11** [comment]
Closing as a duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/15261. The issue has been fixed in the unstable build, and the fix will be included in the next update. Sorry for the inconvenience!
